docs/getting-started/connect-widget.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Connect Widget"
+description: "Overview of the configurations for the widget."
+---
+
+# Link the the Metriport Connect widget in your app
+
+With Metriport, you get a pre-built [Connect widget](https://connect.metriport.com/?token=demo)
+that you can easily plug into your app, so that your users can connect all of their data sources to your application!
+Here's a preview:
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo"
+>
+  {" "}
+</iframe>
+<br />
+
+# Url Paramaters
+
+Set url params to validate the current session of the widget and make
+certain customizations to fit your product needs.
+
+### token
+
+This is the token generated by metriport. Check the quick start for more details.
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}`;
+```
+
+### colorMode
+
+You can update the color mode to be either light or dark.<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&colorMode=dark`;
+```
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo&colorMode=dark"
+>
+  {" "}
+</iframe>
+
+### customColor
+
+You can set your own color as the highlight color by adding a default color such as orange or add a hex code.<br/>
```

Comment:
Perhaps link to where these color names are coming from? ie is `orange` from CSS? What version?